src/Console/ModelsCommand.php

```diff
@@ -810,12 +814,13 @@ protected function createPhpDocs($class)
 
 
         if ($this->write) {
+            $modelDocComment = $this->write_mixin ? "/**\n * @mixin IdeHelper{$classname}\n */" : $docComment;
```

Comment:
This will replace the existing attributes, right? Not sure if we want that?

Reply:
You are correct, I did not think about this, we should keep the exisiting ones, will look into a way to do that.